DB/projects.json

```
@@ -3472,6 +3472,12 @@
         "title": "Number To Words Convertor",
         "description": "This project can convert the entered number into international system of numbering instantly.",
         "tech": ["html", "css", "JAVASCRIPT"]
+      },
+      {
+        "link": "https://github.com/Avdhesh-Varshney/TimeWatcher",
+        "title": "Time Watcher",
+        "description": "This project can tell you about how much time has gone from the entered date and time even in milli-seconds.",
+        "tech": ["HTML", "CSS", "JAVASCRIPT"]
```

convert to lowercase 
```suggestion
        "tech": ["html", "css", "javascript"]
```